Tests/TensorFlowTests/LayerTests.swift

```diff
@@ -361,6 +361,20 @@ final class LayerTests: XCTestCase {
         XCTAssertEqual(output, expected)
     }
 
+    func testDropout() {
```

Comment:
I'd suggest using `withLearningPhase(_:_:)` in tests to make scoping clear and prevent global effects.